README.md

```diff
@@ -1,42 +1,76 @@
+<!-- HTML Graphic image –>
 <p align="center">
   <a href="https://www.rosetta-api.org">
     <img width="90%" alt="Rosetta" src="https://www.rosetta-api.org/img/rosetta_header.png">
   </a>
 </p>
+<!-- HTML h3 Title and Description -->
 <h3 align="center">
    Rosetta Ethereum
 </h3>
+<p align="center">
+This repository contains a sample implementation of Rosetta API for the Ethereum blockchain.
+</p>
+<!-- Badges -->
 <p align="center">
```

Comment:
Could you remove this circleci badge from here. we have moved to github actions. maybe we can find a github actions badge and replace this.

Reply:
This may not be the only repo with that badge. We'll need to check the rest.

Reply:
yes. let's start changing from this repo and others can follow

Reply:
Fixing in next commit.